libpod/container_api.go

```diff
@@ -98,6 +99,12 @@ func (c *Container) Start(ctx context.Context, recursive bool) error {
 	return c.start()
 }
 
+// Update updates the given container.
+// only the cgroup config can be updated and therefore only a linux resource spec is passed.
+func (c *Container) Update(res *spec.LinuxResources) error {
+	return c.update(res)
```

Comment:
Should lock and check status here (use `c.syncContainer()` instead of `c.UpdateContainer`, it handles state transitions properly)